### DISCUSSION TITLE
ScriptReference/GameObject.CreatePrimitive

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#22] [Notes]
# ScriptReference/GameObject.CreatePrimitive

https://docs.unity3d.com/ScriptReference/GameObject.CreatePrimitive.html

**Comment by vertxxyz** [comment]
If you just want primitive **meshes** you can call:
```csharp
Resources.GetBuiltinResource<Mesh>(string path);
```
| Type     | Path                 |
| -------- | -------------------- |
| Sphere   | `"New-Sphere.fbx"`   |
| Capsule  | `"New-Capsule.fbx"`  |
| Cylinder | `"New-Cylinder.fbx"` |
| Cube     | `"Cube.fbx"`         |
| Plane    | `"New-Plane.fbx"`    |
| Quad     | `"Quad.fbx"`         |
